curriculum/challenges/english/01-responsive-web-design/basic-css/use-attribute-selectors-to-style-elements.english.md

```diff
@@ -31,7 +31,7 @@ Using the <code>type</code> attribute selector, try to give the checkboxes in Ca
 ```yml
 tests:
   - text: The <code>type</code> attribute selector should be used to select the checkboxes.
-    testString: assert(code.match(/<style>[\s\S]*?\[type=("|')checkbox\1\]\s*?{[\s\S]*?}[\s\S]*?<\/style>/gi));
+    testString: assert(code.match(/<style>[\s\S]*?\[type[\s\S]*?=[\s\S]*?("|')checkbox\1\]\s*?{[\s\S]*?}[\s\S]*?<\/style>/gi));
```

Comment:
@jaredwils Unfortunately, the regex you have suggested would allow any character after type. For example, the following would pass incorrectly:
```
[type == 'checkbox']
```
```
[types='checkbox]
```
I suggest just allowing whitespace:
```suggestion
    testString: assert(code.match(/<style>[\s\S]*?\[type\s*=\s*("|')checkbox\1\]\s*?{[\s\S]*?}[\s\S]*?<\/style>/gi));
```